test/index.test.js

```diff
@@ -339,6 +345,7 @@ describe('index', function () {
         it('sets the memory appropriately when ram is set to HIGH', () => {
             postConfig.json.metadata.cpu = 2000;
             postConfig.json.metadata.memory = 12;
+            fakeStartConfig.annotations['beta.screwdriver.cd/cpu'] = 'LOW';
```

Comment:
`LOW` setting is extra, because it is default setting. The same applies to the following test cases.

Reply:
There is no need to set default `LOW`. I remove those. Thank you.